src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs

```diff
@@ -526,6 +518,48 @@ private void StartTestRunAttachmentsProcessing(TestRunAttachmentsProcessingPaylo
                 });
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="payload"></param>
+        /// <param name="testRequestManager"></param>
+        private void StartTestRunner(StartTestRunnerPayload payload, ITestRequestManager testRequestManager)
+        {
+            Task.Run(
+                delegate
```

Comment:
Is there a reason this is delegate and not lambda `() =>`? I never saw it used like this.

Reply:
Just for the sake of consistency, this is the way to start a Task the rest of the methods in this file adhere to.